.github/workflows/pr-tests.yml

```
@@ -0,0 +1,204 @@
+    
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20.11'
+        cache: 'npm'
+    
+    - name: Cache dependencies
+      uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)